src/js/constants/NameMappings.js

```diff
@@ -0,0 +1,13 @@
+var AppStatus = require("../constants/AppStatus");
+
+const NameMappings = {
+  statusNameMapping: {
```

Comment:
It's because I have a stupid opinion on having an own file for statusNameMapping only.
I don't like this somehow, because it actually a view thing and not a real constants file.
With a general file called NameMappings I hide this away and fool myself.
And possibly it's growing, yes, but that it's not an argument.
Should I rename it?